### PR TITLE
several actor funcs

### PR DESCRIPTION
**Pull request by HailSanta**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 8 functions (+0.17%) on `us`.

File: ver/us/splat.yaml [hunk 1]
```diff
@@ -4163,7 +4163,6 @@ segments:
     vram: 0xE0120000
     subsegments:
     - [0x40B3F0, c, sun]
-    - [0x40BB70, data]
```
Comment (ethteck) [inline ver/us/splat.yaml:4166]: for the sake of record keeping/whatever, we tend to leave these as `- [0x40BB70]`

**Review by ethteck — CHANGES_REQUESTED** [review]
gg

**Review by ethteck — APPROVED** [review]

File: src/182B30.c [hunk 2]
```diff
@@ -785,7 +942,55 @@ void update_player_actor_shadow(void) {
...
+void func_80258E14(void* arg0) {
+    Matrix4f mtxRotX, mtxRotY, mtxRotZ, mtxRotation, mtxScale;
+    Matrix4f mtxPivotOn, mtxPivotOff, mtxTranslate;
+    Matrix4f mtxTemp, mtxTransform, mtxMirror;
+    Actor* player = gBattleStatus.playerActor;
+    ActorPart* part = &player->partsTable[0];
+    f32 playerYaw;
+    f32 dx, dy, dz;
+
+    playerYaw = player->yaw;
```
Comment (nanaian) [inline src/182B30.c:954]:
```suggestion
    f32 playerYaw = player->yaw;
    f32 dx, dy, dz;

```

File: src/182B30.c [hunk 3]
```diff
@@ -1901,43 +2108,192 @@ s32 func_8025C840(s32 arg0, ActorPart* part, s32 yaw, s32 arg3) {
...
+    }
+    
+    decor->unk75C += 7;
+    baseAngle = decor->unk75C;
+    
+    if (decor->unk75C >= 360) {
+        decor->unk75C = baseAngle - ((baseAngle / 360) * 360);
+    }
+    
+    for (i = 0; i < 20; i++) {
```
Comment (nanaian) [inline src/182B30.c:2151]:
```suggestion
    for (i = 0; i < ARRAY_COUNT(rbuf); i++) {
```

File: src/182B30.c [hunk 4]
```diff
@@ -1901,43 +2108,192 @@ s32 func_8025C840(s32 arg0, ActorPart* part, s32 yaw, s32 arg3) {
...
+    
+    alpha = 255;
+    if (part->opacity < 255) {
+        alpha = part->opacity;
+    }
+    if (part->flags & 0x100) {
+        alpha = (alpha * 120) / 255;
+    }
+    
+    for (i = 0; i < 20; i++) {
```
Comment (nanaian) [inline src/182B30.c:2165]:
```suggestion
    for (i = 0; i < ARRAY_COUNT(rbuf); i++) {
```

File: src/182B30.c [hunk 5]
```diff
@@ -1901,43 +2108,192 @@ s32 func_8025C840(s32 arg0, ActorPart* part, s32 yaw, s32 arg3) {
...
+            break;
+        case 1:
+            decor->unk_767 = 8;
+            decor->unk_766 = 1;
+            break;
+        case 2:
+        default:
+            decor->unk_767 = 0xE;
+            decor->unk_766 = 2;
+            break;
```
Comment (nanaian) [inline src/182B30.c:2219]: Indent switch contents pls

**Review by nanaian — CHANGES_REQUESTED** [review]

**Pull request merged**